app/controllers/api/V1/appointments_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Api::V1::AppointmentsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    appointments = @user.appointments
+    render json: { status: 'Success', message: 'Appointments fetched successfully', data: appointments }, status: :ok
+  end
```

Comment:
- [ ] Here, when I tried to fetch a user that did not exist in the database I'm getting an error from the API: 

![Captura de Pantalla 2022-03-26 a la(s) 20 53 44](https://user-images.githubusercontent.com/88604987/160265168-b8053e2f-c124-4437-9156-92bc77e67c1a.png)
You can easily fix adding a condition something like this:
@user = User.find_by_id(params[:user_id])
if @user.nil?
  render json: { status: 'error', message: 'Appointments for this user do not exist' }
else 
  render json: { status: 'success', message: 'Appointments fetched successfully', data: appointments }, status: :ok
end

Reply:
Note: I'm using `find_by_id` instead `find`, with this little change I'm able to use `.nil?` to check if my database query is nil instead of an active record error because can't find a user with the id provided.